influxdb/tests/py26_test.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
+
+import sys
+import os
+import datetime
+import requests
+import time
+
+from influxdb import InfluxDBClient
+
+# InfluxDB Server parameters
+influxdb_server = "localhost"
+port = 8086
+database_name = "test_db"
+ip_addr = os.popen("hostname").read()
+
+# InfluxDB connection and database creation if it doesn't exists
+client = InfluxDBClient(influxdb_server, port, "influx-user", "influx-pass")
+
+try:
+    client.create_database(database_name)
+except requests.ConnectionError as e:
+    print("Failed to start, unable to establish connection with InfluxDB server %s - %s" % (influxdb_server, e))
+    sys.exit(2)
+
+# System Load
+load_avg_1 = os.popen("cat /proc/loadavg|awk '{print $1,$2,$3}'").read()[0]
```

Comment:
Can you please look to use the `subprocess.check_output` method here? The `os.popen` commands have been deprecated.

https://docs.python.org/2/library/subprocess.html#subprocess.check_output
https://docs.python.org/2/library/subprocess.html#replacing-os-popen-os-popen2-os-popen3

Reply:
unfortunately, check_output is not available on python2.6